jsonnet/kube-thanos/kube-thanos-rule.libsonnet

```diff
@@ -150,7 +166,8 @@ function(params) {
             labels: tr.config.commonLabels,
           },
           spec: {
-            containers: [c],
+            containers: [c] + 
+              (if std.length(tr.config.rulesConfig) > 0 then [reloadC] else []),
```

Comment:
Is there any reason not to simply always add this?

Reply:
I do think that we need to add this container only if we have `rulesConfig` defined. if there is not `--rule-file` defined, we do not need to have this feature. WDYT?